src/utils/openmc/statepoint.py

```diff
@@ -7,6 +8,9 @@
 import openmc
 from openmc.constants import *
 
+if sys.version > '3':
```

Comment:
For the record, I use Python 3 exclusively these days.  So you have at least one user :).  Adding Python 3 to Travis would probably help.